recipe/build.sh

```diff
@@ -1,18 +1,17 @@
 mkdir -p build
 cd build
 
-if [ `uname` = "Darwin" ]; then
-      cpp_std=14
-else
-      cpp_std=17
-fi
+# if [ `uname` = "Darwin" ]; then
+#       cpp_std=14
+# else
+#       cpp_std=17
+# fi
 
 cmake -G "Ninja" \
+      -D CMAKE_BUILD_TYPE:STRING="Release" \
       -D CMAKE_INSTALL_PREFIX:FILEPATH=$PREFIX \
-      -D CMAKE_BUILD_TYPE=Release \
       -D CMAKE_PREFIX_PATH:FILEPATH=$PREFIX \
-      -D CMAKE_SYSTEM_PREFIX_PATH:FILEPATH=$PREFIX \
-      -D CMAKE_CXX_STANDARD=${cpp_std} \
+      -D Boost_INCLUDE_DIR:FILEPATH=$PREFIX/include/boost \
```

Comment:
This could solve your problem:
```suggestion
      -DBoost_NO_BOOST_CMAKE=ON \
```

Reply:
thanks for the hint!